cfme/services/myservice/ui.py

```diff
@@ -263,7 +263,7 @@ def retire(self):
             num_sec=10 * 60, delay=3,
             message='Service Retirement wait')
     else:
-        request_descr = "Provisioning Service [{0}] from [{0}]".format(self.name)
+        request_descr = f"Service Retire for: {self.name}"
```

Comment:
_Optional_ consider changing this text to

```suggestion
request_descr = f"Retiring service: {self.name}"
```